### ISSUE TITLE
embeddings

### ISSUE THREAD

**Issue by jwijffels** [EmilHvitfeldt/textrecipes#10]
As you probably plan to add embeddings to this functionality, you might be interested to also look at https://github.com/bnosac/ruimtehol
It does all kinds of embeddings (words/sentences/articles/labels/graphs/...). It might enrich the way you think of possible recipe steps. And while you are looking at at, would appreciate any feedbacks you have on the package :) before I plan to release it to cran

**Comment by EmilHvitfeldt** [comment]
It looks very interesting! Does it come with pretrained word embeddings?

**Comment by jwijffels** [comment]
No, you can train it on your own data, Wikipedia if you like or start the training from already pre-trained embeddings from another tool.
The predict functionality provides prediction of classes, full text or any kind of labels.
